### ISSUE TITLE
Option to not generate source maps

### ISSUE THREAD

**Issue by timothyjensen** [craigsimps/gulp-wp-toolkit#95]
It would be nice if to have some basic configurability of when/where to use source maps.

## Context
In my `gulpfile.js` I have set up two stylesheets, `style.css` and `style.min.css`. My theme enqueues `style.min.css` unless `SCRIPT_DEBUG` is not null. This allows me to work with the expanded stylesheet during development, which is the only time that I want source maps to load.  I would love to be able to remove source maps from `style.min.css` so that they do not load on production sites.

In short, what I'm looking to do is to have source maps for `style.css` but no source maps for `style.min.css`.
